README.md

```diff
@@ -1,9 +1,9 @@
 # PKL
 _________________________________________
 Latihan dan project selama PKL di UrRemote
-Dari tgl 08 July 2019 s/d 5 Oktober 2019
+Dari tgl 08 Juli 2019 s/d 5 Oktober 2019
 ________________________________________________
 Folder : Project Html,CSS,JS ==> Terdapat 2 file Html yaitu : volum.html dan Profile.html 3 Folder gambar,js,css.
 
-Folder : Project Ruby 1 ==> Terdapat 1 file Hiung.rb file Hiung.rb berisikan 2 class yang dimuat didalamnya yaitu Aritmatika dan Luas Bangun Datar.
+Folder : Project Ruby 1 ==> Terdapat 1 file Hiung.rb,file Hiung.rb berisikan 2 class yang dimuat didalamnya yaitu Aritmatika dan Luas Bangun Datar.
```

Comment:
Jangan lupa
- ubah `Hiung.rb` jadi `hitung.rb`
- `Folder:` instead of `folder :`